change/@fluentui-eslint-plugin-912d0763-d6cf-4fec-beac-2843c49fc62d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31988&commitId=ec2c6b5cf058473051aa64f859dc171099502721&env=prod&clientType=FLUENTUI)


  <details><summary><b>Checkbox</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Circular Indeterminate - Dark Mode.indeterminate.chromium.png|0| Removed|
|Checkbox.Circular Indeterminate.indeterminate.chromium.png|0| Removed|
|Checkbox.Indeterminate - Dark Mode.indeterminate.chromium.png|0| Removed|
|Checkbox.Indeterminate.indeterminate.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>